gammapy/astro/darkmatter/spectra.py

```diff
@@ -29,6 +29,36 @@ class PrimaryFlux(object):
     ----------
     * `2011JCAP...03..051 <http://adsabs.harvard.edu/abs/2011JCAP...03..051>`_
     """
+    channel_registry = {
+        'eL' : 'eL',
+        'eR' : 'eR',
+        'e' : 'e',
+        'muL' : '\\[Mu]L',
+        'muR' : '\\[Mu]R',
+        'mu' : '\\[Mu]',
+        'tauL' : '\\[Tau]L',
+        'tauR' : '\\[Tau]R',
+        'tau' : '\\[Tau]',
+        'q' : 'q',
+        'c' : 'c',
+        'b' : 'b',
+        't' : 't',
+        'WL' : 'WL',
+        'WT' : 'WT',
+        'W' : 'W',
+        'ZL' : 'ZL',
+        'ZT' : 'ZT',
+        'Z' : 'Z',
+        'g' : 'g',
+        'gamma' : '\\[Gamma]',
+        'h' : 'h',
+        'nu_e' : '\\[Nu]e',
+        'nu_mu' : '\\[Nu]\\[Mu]',
+        'nu_tau' : '\\[Nu]\\[Tau]',
+        'V->e' : 'V->e',
+        'V->mu' : 'V->\\[Mu]',
+        'V->tau' : 'V->\\[Tau]',
+    }
```

Comment:
I think if you add a string after the class-level dict, it will show up as docstring in the docs.

Reply:
I would prefer no to show this dict in the docs, in order to now confuse anyone with the two sets of channel names